addons/mail/tests/test_link_preview.py

```diff
@@ -161,10 +161,6 @@ def get_bus_params():
 
             with self.assertBus(get_params=get_bus_params):
                 self.env["mail.link.preview"]._create_from_message_and_notify(message)
-            link_preview_count = self.env["mail.link.preview"].search_count(
-                [("source_url", "=", self.source_url)]
-            )
-            self.assertEqual(link_preview_count, throttle + 1)
```

Comment:
The throttle was removed before this PR when the code was refactored by @tde-banana-odoo in https://github.com/odoo-dev/odoo/commit/84e6596fa1901b90874fb18020e8b687209e4334#diff-f3adff47adc333fc6613f301cb0a0629d8dea23cf5cf0926dea4a8ab05ba1199R40

This PR also makes the throttle useless since we only create the link_preview once. Therefore, we only make one request for every URL and reuse it.